check-node-version.js

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env node
+
+const minimumVersion = '12.0.0';
+const currentVersion = process.version.replace('v', '');
+
+const minimumVersionMajor = parseInt(minimumVersion.split('.')[0]);
```

Comment:
Why not just do:

```suggestion
const minimumVersionMajor = 12
```

Reply:
Not a blocker btw, just seems simpler to declare the variable as what we want (rather than dynamically building it).